### ISSUE TITLE
Error while installing stellar core

### ISSUE THREAD

**Issue by sahar-fehri** [stellar/stellar-core#2067]
Hello everyone ,

I am trying to follow this tutorial in order to build a private stellar network.
https://labs.imaginea.com/post/stellar-bc-wallet/
And i am still trying to install stellar-core.
when i ran ./configure this is the error i get :
./configure: line 17739: syntax error near unexpected token `libsodium,'
./configure: line 17739: `PKG_CHECK_MODULES(libsodium, libsodium >= 1.0.13, :, libsodium_INTERNAL=yes)'

Anyone has any idea how to solve it ?  i made sure that i upgraded my gcc compiler to version6 but im still struggling with the installation.

Thank you,

**Comment by shotcollin** [comment]
Although I realize this issue is closed and you have a limited community to respond to issues, I wanted to report that I got the same error attempting a build on OSX (10.13.6), having installed all the listed dependencies at https://github.com/stellar/stellar-core/blob/master/INSTALL.md#OSX

Same error as above:
./configure: line 17696: syntax error near unexpected token 'libsodium,'  
./configure: line 17696: `PKG_CHECK_MODULES(libsodium, libsodium >= 1.0.13, :, libsodium_INTERNAL=yes)'

Commenting out that line of the configure script causes it to get to here:  
./configure: line 17728: syntax error near unexpected token 'libsodium,'  
./configure: line 17728: '   PKG_CHECK_MODULES(libsodium, libsodium)'

Commenting out that line of the configure script causes it to get to here:  
./configure: line 17772: syntax error near unexpected token 'xdrpp,' 
./configure: line 17772: '   PKG_CHECK_MODULES(xdrpp, xdrpp)'

Then here: 
./configure: line 17847: syntax error near unexpected token 'libmedida,' 
./configure: line 17847: 'PKG_CHECK_MODULES(libmedida, libmedida)'  

Then:  
./configure: line 17876: syntax error near unexpected token 'libpq,' 
./configure: line 17876: 'PKG_CHECK_MODULES(libpq, libpq, have_postgres=1)'

Thinking this might be related to the suggestion at https://github.com/stellar/stellar-core/blob/master/INSTALL.md#OSX to get around configure errors by setting PKG_CONFIG_PATH='/usr/local/lib/pkgconfig', I tried that as well, but no luck.

In short, configure seems to complain about the syntax every time PKG_CHECK_MODULES is called. Having commented out all lines referred to above, `./configure` completes successfully, as does `make`, `make check`, and `make install`, and stellar-core appears to be running properly.

**Comment by nektro** [comment]
this issue is that `pkg-config`/`pkgconf` was not installed, specifically that pkg-config/pkgconf's `share/aclocal` folder needs to be visible to `ACLOCAL_PATH`
